shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/data/repository/SurveyRepositoryImpl.kt

```diff
@@ -20,4 +22,9 @@ class SurveyRepositoryImpl(private val networkDataSource: NetworkDataSource): Su
                 )
             }
     }
+
+    override fun surveyDetail(id: String): Flow<SurveyDetail> {
```

Comment:
@blyscuit just a suggestion, should we put the prefix with `action`? such as `getSurveyDetail`, `featchSuveryDetail`,etc...? Using `surveyDetail` make me confused tbo 🙏

Reply:
Good catch, I will modify this repository. 🙏